index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+  <meta charset="UTF-8">
+  <title>JavaScript課題②</title>
+  <style>
+    /* .Num {
+      float: left;
+      clear: both;
+    } */
+    li {
+      list-style: none;
+    }
+  </style>
+</head>
+
+<body>
+  <h1>FizzBuzz問題</h1>
+  <div class="subtitle">FizzNum:
+    <input id="fizzNum" class="Fizz Num" type="text" placeholder="整数値を入力してください" style="width: 170px;">
+
+  </div>
+  <div class="subtitle">BuzzNum:
+    <input id="buzzNum" class="Buzz Num" type="text" placeholder="整数値を入力してください" style="width: 170px;">
+
+  </div>
+  <div>
+    <input id="button" type="submit" value="実行">
```

Comment:
名前の抽象度が高く、コードを見た人が
「この HTML 要素がどんな役割を持っているのか」が理解しにくくなってしまってます。
もう少し名前に気を使ってほしいです。

```suggestion
    <input id="runButton" type="submit" value="実行">
```